OpenRA.Mods.Common/Traits/World/EditorResourceLayer.cs

```diff
@@ -182,6 +182,26 @@ void UpdateNetWorth(string oldResourceType, int oldDensity, string newResourceTy
 				NetWorth += (newDensity + 1) * newResourceValue;
 		}
 
+		public int CalculateRegionValue(CellRegion sourceRegion)
+		{
+			var resourceValueInRegion = 0;
+			foreach (var cell in sourceRegion)
+			{
+				var mcell = cell.ToMPos(Map);
+				if (Map.Resources.Contains(mcell) && Map.Resources[mcell].Type != 0)
+				{
+					resourceValueInRegion++;
+					var rcell = Map.Resources[mcell];
+					if (ResourceTypesByIndex.TryGetValue(rcell.Type, out var resourceType) && resourceValues.TryGetValue(resourceType, out var resourceValuePerUnit))
+					{
+						resourceValueInRegion += Tiles[mcell].Density * resourceValuePerUnit;
+					}
```

Comment:
```suggestion
						resourceValueInRegion += Tiles[mcell].Density * resourceValuePerUnit;
```

style nit: you should have braces around single line conditionals and loops